### ISSUE TITLE
Changing columns to display

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#369]
_Imported from trac ticket 369.
Created by **jneville**
Opened in trac: Oct 26, 2010, 11:37:30 AM
Last modified in trac: Jun 16, 2014, 9:29:05 AM_
***
_by jneville:_
Some day it would be nice to be able to add and delete the columns displayed on orders, much like most email programs allow. Should consider any other column displays too, ie products

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 10, 2013, 5:14:10 PM:_

Ticket [#978](978) has been marked as a duplicate of this ticket.

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 10, 2013, 5:17:41 PM:_

Will depend on #201

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 16, 2013, 8:10:55 PM:_

(Changeset [[4668]](../commit/535f5c37edff22660b21041919ab965d52d2a6cf))
* re #369, add RDataGrid framework to be able to hide/reorder grid columns.  User interface not available until #369 #all

**Comment by kentbower** [comment]
_**kbower** trac comment on Oct 28, 2013, 9:21:03 AM:_

Calvin, if you'd like a small tutorial on the framework that exists, just let me know.

**Comment by kentbower** [comment]
_**criley** trac comment on Jun 16, 2014, 9:29:05 AM:_

(Changeset [[5457]](../commit/68a391c2228e3f42f771726bafa611c939faa0ee))
* add ability to save grid column order as user's preference (grid right-click) #all
